ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/ClusterPageDetails.tsx

```diff
@@ -1,18 +1,119 @@
+/* eslint-disable no-nested-ternary */
 import React from 'react';
-import { PageSection, Text } from '@patternfly/react-core';
+import {
+    Bullseye,
+    Card,
+    CardBody,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Flex,
+    PageSection,
+    Spinner,
+    Text,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
 
-// eslint-disable-next-line @typescript-eslint/ban-types
-export type ClusterPageDetailsProps = {};
+import EmptyStateTemplate from 'Components/EmptyStateTemplate';
+import { getDateTime } from 'utils/dateUtils';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import ExpandableLabelSection from '../../components/ExpandableLabelSection';
+import useClusterExtendedDetails from './useClusterExtendedDetails';
+import { displayClusterType } from '../utils/stringUtils';
+
+export type ClusterPageDetailsProps = {
+    clusterId: string;
+};
+
+function ClusterPageDetails({ clusterId }: ClusterPageDetailsProps) {
+    const { data, loading, error } = useClusterExtendedDetails(clusterId);
 
-// eslint-disable-next-line no-empty-pattern
-function ClusterPageDetails({}: ClusterPageDetailsProps) {
     return (
         <>
             <PageSection component="div" variant="light" className="pf-v5-u-py-md pf-u-px-xl">
                 <Text>View details about this cluster</Text>
             </PageSection>
             <PageSection isFilled className="pf-v5-u-display-flex pf-v5-u-flex-direction-column">
-                <div className="pf-v5-u-flex-grow-1 pf-v5-u-background-color-100">Details</div>
+                <Card>
+                    <CardBody>
+                        {error ? (
+                            <Bullseye>
+                                <EmptyStateTemplate
+                                    title="There was an error loading the cluster details"
+                                    headingLevel="h2"
+                                    icon={ExclamationCircleIcon}
+                                    iconClassName="pf-v5-u-danger-color-100"
+                                >
+                                    {getAxiosErrorMessage(error)}
+                                </EmptyStateTemplate>
+                            </Bullseye>
+                        ) : loading ? (
+                            <Bullseye>
+                                <Spinner size="xl" />
+                            </Bullseye>
+                        ) : (
+                            data && (
```

Comment:
Is this an example of your recent comment about objects in conditional rendering?
https://github.com/stackrox/stackrox/pull/10959#discussion_r1586438310

Reply:
Great eyes! It seems to me this will cause a problem.

I just merged the react types PR, let me rebase and see what happens (but fix this either way for consistency.)

Reply:
Ok, so this doesn't cause an issue after rebase. The reason is because in the few cases that were fixed in the comment link above, the value of `error` was `unknown`, and in this case (and likely others) the value is a more concrete known type.